talisker/testing.py

```diff
@@ -71,7 +72,8 @@ def temp_file():
 
 def clear_all():
     """Clear all talisker state."""
-    talisker.context.clear()  # talisker request_context
+    Context.clear()  # talisker request_context
```

Comment:
why? - doesn't CONTEXTS.clear() below clear everything

Reply:
It does, will fix